libs/styles/styles.css

```diff
@@ -223,3 +223,18 @@ main > .section > .content {
   max-width: var(--grid-container-width);
   margin: 0 auto;
 }
+
+div[data-failed=true] {
+  border: 1px solid red;
+  border-top-width: 20px;
+  position: relative;
+}
+
+div[data-failed=true]::before {
+  content: attr(data-reason);
+  color: white;
+  position: absolute;
+  top: -20px;
+  font-size: 14px;
+  font-weight: 600;
+}
```

Comment:
Small styling suggestions.

A) They look better on mobile.
B) They fit more with Milo's brand.

![Screen Shot 2022-08-09 at 7 59 02 PM](https://user-images.githubusercontent.com/1972095/183793801-c7dcc3e6-647d-46fe-a08c-c6d49a2b2dd9.png)

```suggestion
div[data-failed=true] {
  border: 4px solid #ff00a0;
  position: relative;
  margin: 12px;
  border-radius: 6px;
  display: block;
}

div[data-failed=true]::before {
  content: attr(data-reason);
  color: white;
  font-weight: 600;
  padding-left: 8px;
  background: #ff00a0;
  line-height: 28px;
  display: block;
  padding-bottom: 6px;
}
```